pages/portfolio.html

```diff
@@ -4,9 +4,82 @@
+    <section class="description left-column">
+      <h3>Task List API</h3>
+      <p>A web API created to organize tasks and goals, with an integrated slackbot. <a href="https://ada-jfd-task-list-api.herokuapp.com/tasks">Tasks</a> can be sorted, filtered, marked complete, and added to <a href="https://ada-jfd-task-list-api.herokuapp.com/goals">goals</a>. Project deployed on Heroku.</p>
+      <h4>Skills Assessed</h4>
+      <ul>
+        <li>Demonstrating understanding of the client-server model, request-response cycle and conventional RESTful routes</li>
+        <li>Reading and using existing external web APIs</li>
+        <li>Creating databases and models</li>
+        <li>Create conventional RESTful routes as well as unconventional routes for custom behavior</li>
+        <li>Creating a one-to-many relationship between two models</li>
+        <li>Writing tests</li>
+        <li>etc</li>
+      </ul>
+    </section>
+    <div class="git-link right-column"><a href="https://github.com/jfdudley/task-list-api"><img src="../images/noun-task-list-2154088.png" alt="checklist icon"></a></div> 
+    <div class="git-link left-column"><a href="https://github.com/jfdudley/solar-system-api"><img src="../images/noun-solar-system-3328563.png" alt="solar system icon"></a></div> 
+    <section class="description right-column">
+      <h3>Solar System API</h3>
+      <p>A web API created via pair programming to track <a href="https://ada-jfd-solar-system-api.herokuapp.com/planets">planets</a>, their associated moons, and other relevant data. Project deployed on Heroku.</p>
+      <h4>Skills Assessed</h4>
+      <ul>
+        <li>Demonstrating understanding of the client-server model, request-response cycle and conventional RESTful routes</li>
+        <li>Pair programming skills</li>
+        <li>Creating relational databases and models</li>
+        <li>Creating a one-to-many relationship between two models</li>
+        <li>Writing tests</li>
+        <li>etc</li>
+      </ul>
+    </section>
+    <section class="description left-column">
+      <h3>Swap Meet</h3>
+      <p>A program to track vendors and their respective itineraries, and allow for exchanging of items between vendors based on various criteria.</p>
+      <h4>Skills Assessed</h4>
+      <ul>
+        <li>Creating multiple classes, using both composition and inheritance</li>
+        <li>Writing instance methods</li>
+        <li>Importing code from other modules</li>
+        <li>Practice writing DRY code</li>
+        <li>Practice good Git hygiene</li>
+        <li>Writing tests</li>
+        <li>etc</li>
+      </ul>
+    </section>
+    <div class="git-link right-column"><a href="https://github.com/jfdudley/swap-meet"><img src="../images/noun-swap-3401618.png" alt="swap icon"></a></div> 
+    <div class="git-link left-column"><a href="https://github.com/jfdudley/viewing-party"><img src="../images/noun-watching-movie-4224361.png" alt="movie watching icon"></a></div> 
```

Comment:
Similar to the feedback on line 26, it would help readability to break these image blocks across several lines:
```html
<div class="git-link left-column">
    <a href="https://github.com/jfdudley/solar-system-api">
        <img src="../images/noun-solar-system-3328563.png" alt="solar system icon">
    </a>
</div>
```